config/environments/test.rb

```diff
@@ -52,6 +52,8 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
 
+  config.active_job.queue_adapter = :test
```

Comment:
By enabling the test adapter globally we ensure there's no risk of brittle between using `:inline` (which mostly should be never used) and `:test` for tests involving ActiveJob